lib/timely/rails/extensions.rb

```diff
@@ -1,55 +1,30 @@
+# frozen_string_literal: true
+
 module Timely
   module Extensions
     # Add a WeekDays attribute
     #
     # By default it will use attribute_bit_array as db field, but this can
     # be overridden by specifying :db_field => 'somthing_else'
-    def weekdays_field(attribute, options={})
+    def weekdays_field(attribute, options = {})
       db_field = options[:db_field] || attribute.to_s + '_bit_array'
-      self.composed_of(attribute,
-        :class_name => "::Timely::WeekDays",
-        :mapping    => [[db_field, 'weekdays_int']],
-        :converter  => Proc.new {|field| ::Timely::WeekDays.new(field)}
-      )
+      composed_of(attribute,
+                  class_name: '::Timely::WeekDays',
+                  mapping: [[db_field, 'weekdays_int']],
+                  converter: proc { |field| ::Timely::WeekDays.new(field) })
     end
 
     def acts_as_seasonal
-      belongs_to :season, :class_name => 'Timely::Season', optional: true
+      belongs_to :season, class_name: 'Timely::Season', optional: true
       accepts_nested_attributes_for :season
       validates_associated :season
 
-      if ::ActiveRecord::VERSION::MAJOR >= 3
-        scope :season_on, lambda { |*args|
```

Comment:
Do we still need these scopes?

Reply:
Not as far as I can see. QT actually has long had its similar scopes - see FareBasis.overlapping. Note FareBasis is the only acts_as_seasonal model, afaik.